gormigrate.go

```diff
@@ -157,12 +157,12 @@ func (g *Gormigrate) migrate(migrationID string) error {
 		return err
 	}
 
+	g.begin()
```

Comment:
Perhaps we could do

```go
defer g.rollback()
```

right on the next line and remove other calls to `g.rollback()`? We don't check for error on rollback, so this is a no-op.